npm_and_yarn/helpers/lib/npm/conflicting-dependency-parser.js

```diff
@@ -15,6 +15,8 @@ const semver = require("semver");
 async function findConflictingDependencies(directory, depName, targetVersion) {
   const arb = new Arborist({
     path: directory,
+    dryRun: true,
```

Comment:
I don't think these do anything for this usage of Arborist but am setting them for consistency.